### DISCUSSION TITLE
windowsInit sqflite_common_ffi implementation

### DISCUSSION THREAD

**Discussion by Taosif7** [tekartik/sqflite#568] [General]
I've been building desktop (windows) version of my flutter application and I'm encountering this error :  
```
Unhandled Exception: MissingPluginException(No implementation found for method getDatabasesPath on channel com.tekartik.sqflite)
```

With some research and reading issues, I found out that the sqflite_common_ffi windows implementation, has a method for getting sqlite3.dll file which is causing error. this is because it cannot find the path to `sqlite3.dll`, I also found out an issue that mentions that the `windowsInit` method is only implemented as reference.  
I also followed some suggestions from the issue (#547) But the problem persists.  
Can anyone help me to solve this issue.

The link to file where this is:
https://github.com/tekartik/sqflite/blob/develop/sqflite_common_ffi/lib/src/windows/setup.dart

**Comment by alextekartik** [comment]
I think this has been fixed in release mode to look in the current folder. (i.e. copy the dll along with your executable)
